platform/src/main/java/org/hillview/sketches/PrivateHistogram.java

```diff
@@ -0,0 +1,60 @@
+package org.hillview.sketches;
+
+import org.apache.commons.math3.distribution.LaplaceDistribution;
+import org.hillview.dataset.api.IJson;
+
+import java.io.Serializable;
+
+public class PrivateHistogram extends Histogram implements Serializable, IJson {
+    private final DyadicHistogramBuckets dyadicBuckets;
+
+    private final double maxValue;
+    private final double minValue;
+
+    // Confidence intervals
+    private double[] confMins;
+    private double[] confMaxes;
+
+    public PrivateHistogram(final IHistogramBuckets bucketDescription) {
+        super(bucketDescription);
+        dyadicBuckets = (DyadicHistogramBuckets)bucketDescription;
+
+        this.minValue = dyadicBuckets.getMin();
+        this.maxValue = dyadicBuckets.getMax();
+
+        // Not currently used
+        this.confMins = new double[dyadicBuckets.getNumOfBuckets()];
```

Comment:
Do you need to store these or can you just compute them?

Reply:
I think we can just compute them.